### PR TITLE
Addressed issue #360

### PR DESCRIPTION
**Pull request by coverney**
and added test

### CODE REVIEW

File: code/subpopulation_statistics.m [hunk 1]
```diff
@@ -68,4 +68,23 @@
...
 excluded = size(data,1) - sum(counts);
 
+% Iterate through bins and counts to see if detector is saturated
+saturation_threshold = TASBEConfig.get('flow.saturationWarning');
+min_valid_count = TASBEConfig.get('flow.saturationMinValidCount');
+for i=5:nbins
+    avg_counts = mean(counts(i-4:i-2));
+    if avg_counts < (saturation_threshold*counts(i))
+        no_saturation = 0;
+        for j=i+2:nbins
```
Comment (jakebeal) [inline code/subpopulation_statistics.m]: note that this can be done more compactly and efficiently with a find.

Comment (coverney) [reply code/subpopulation_statistics.m]: Good to know! :)

Comment (coverney) [reply code/subpopulation_statistics.m]: Should I make the change this time?

Comment (jakebeal) [reply code/subpopulation_statistics.m]: It would be good code hygiene, I think.  I merged anyway, since it should be functionally the same.

**Pull request merged**